src/v3/src/components/WebAuthNAutofill/WebAuthNAutofill.tsx

```diff
@@ -48,6 +49,10 @@ const WebAuthNAutofill: UISchemaElementComponent<{
           });
         }
       } catch (err) {
+        // iOS Safari throws an empty object when the user does not use passkey autofill
+        if (isEmpty(err)) {
```

Comment:
Should we check for iOS?
https://github.com/okta/okta-signin-widget/blob/master/src/util/BrowserFeatures.ts#L82